cli/src/cmd/forge/test.rs

```diff
@@ -346,6 +348,21 @@ impl TestOutcome {
     }
 }
 
+fn block_summary(duration: Duration, outcome: TestOutcome) {
+    let failed = outcome.failures().count();
+    let result = match failed {
+        0 => Colour::Green.paint("[ OK ]"),
+        _ => Colour::Red.paint("[FAIL]"),
+    };
```

Comment:
more idiomatic to do an `let result = if outcome.failures().count() == 0 {} else {};`here